api_feature.go

```diff
@@ -62,6 +90,9 @@ func (f *APIFeature) RegisterSteps(ctx *godog.ScenarioContext) {
 	ctx.Step(`^the HTTP status code should be "([^"]*)"$`, f.TheHTTPStatusCodeShouldBe)
 	ctx.Step(`^the response header "([^"]*)" should be "([^"]*)"$`, f.TheResponseHeaderShouldBe)
 	ctx.Step(`^I should receive the following response:$`, f.IShouldReceiveTheFollowingResponse)
+	ctx.Step(`^I have a healthcheck interval of (\d+) second$`, f.iHaveAHealthCheckIntervalOfSecond)
```

Comment:
```suggestion
	ctx.Step(`^I have a healthcheck interval of (\d+) seconds?$`, f.iHaveAHealthCheckIntervalOfSecond)
```

allows it to be 1 second, 2 seconds

Reply:
same for others.

Reply:
Done